src/main/groovy/life/qbic/datamodel/accounting/CostItem.groovy

```diff
@@ -0,0 +1,43 @@
+package life.qbic.datamodel.accounting
+
+/**
+ * An item with associated costs and quantity
+ *
+ * Cost items describe services and their price. They are the building blocks of invoices and offers.
+ * The unit price is always provided in euros.
+ * @since: 1.9.0
+ * @author: Tobias Koch
+ *
+ * * */
+class CostItem {
+
+    double quantity
+    final double UNIT_PRICE
+    final String UNIT_NAME
+    final String NAME
+    final String DESCRIPTION
+
```

Comment:
maybe we need a list of packages of which this item consits?

Reply:
I would try to keep it as simple as possible in the beginning. Therefore I tried to avoid the nesting of "packages" and "items".

Reply:
True, but imo it is important to have the connection between the item and the packages